express-app/controllers/user.js

```diff
@@ -84,6 +85,7 @@ module.exports = {
           newUser.save()
           .then((user) => {
             response.render('dashboard', {user: user});
+            Section.getQuestions(user, request, response);
```

Comment:
you render that dashboard view and then ask for the sections? and in the sections controller you render again the dashboard view??